esp_link/esp_link.py

```diff
@@ -52,14 +53,14 @@ def __init__(self, loop, verbose):
         self.qos = config[6]
         # Handle case where ESP8266 has not been initialised to the WLAN
         sta_if = network.WLAN(network.STA_IF)
-        ap = network.WLAN(network.AP_IF) # access-point interface.
-        ap.active(False)         # deactivate AP interface.
+        ap = network.WLAN(network.AP_IF)  # access-point interface.
```

Comment:
Thanks for this, but I have substantially changed this in that I subclass Client and implement bad_wifi. So most of this code is gone.

I suggest holding off any further changes to esp_link.py until I commit a stable version.